interfaces/builtin/home.go

```diff
@@ -63,14 +76,56 @@ owner /run/user/[0-9]*/gvfs/{,**} r,
 owner /run/user/[0-9]*/gvfs/*/**  w,
 `
 
+const homeConnectedPlugAppArmorWithAllRead = `
+# Allow non-owner read to non-hidden and non-snap files and directories
```

Comment:
Is this documented anywhere public? The home interface is probably the most used interface we have.

Reply:
Not yet-- this is new :) I will update the documentation for it, yes.